Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserBus.h

```diff
@@ -260,6 +260,8 @@ namespace AzToolsFramework
                 return SourceFileDetails();
             }
 
+            virtual void CreateNewVariantListRequested(const char* /*fromShaderSourcePath*/) {}
+
```

Comment:
Obviously this shouldn't be here. It's a specific feature that creeps in a generic tool, so I need a suggestion for "dependency inversion".